guides/source/command_line.md

```diff
@@ -376,37 +376,37 @@ $ bin/rails destroy model Oops
       remove      test/fixtures/oops.yml
 ```
 
-Rake
-----
+bin/rails
+---------
 
-Rake is Ruby Make, a standalone Ruby utility that replaces the Unix utility 'make', and uses a 'Rakefile' and `.rake` files to build up a list of tasks. In Rails, Rake is used for common administration tasks, especially sophisticated ones that build off of each other.
+Rake is Ruby Make, a standalone Ruby utility that replaces the Unix utility 'make', and uses a 'Rakefile' and `.rake` files to build up a list of tasks. In Rails, Rake is used for common administration tasks, especially sophisticated ones that build off of each other. Since Rails 5.0+ rake commands implemented in Rails Framework itself. We prefer to use bin/rails instead of rake.
```

Comment:
The Rake explanation feels out of touch and wholly unnecessary. It's an extremely well known tool, so I don't think it's worth. Beginners are meant to use `bin/rails` going forward anyway.